### PR TITLE
Change Chart namespace deployment for https://github.com/Opster/opens…

### PR DESCRIPTION
**Pull request by idanl21**
…earch-k8s-operator/issues/122#issue-1225002998

### CODE REVIEW

File: charts/templates/opensearch-operator-system-namespace.yaml [hunk 1]
```diff
@@ -2,5 +2,4 @@ apiVersion: v1
 kind: Namespace
```
Comment (swoehrl-mw) [inline charts/templates/opensearch-operator-system-namespace.yaml:2]: The namespace object should be deleted completely from the helm chart. Either the user uses an existing namespace or `helm install --create-namespace`.

File: charts/Chart.yaml [hunk 2]
```diff
@@ -15,10 +25,11 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
 appVersion: "1.16.0"
```
Comment (swoehrl-mw) [inline charts/Chart.yaml:33]: I recommend we set this to the version of the operator (so for GA 1.0.0)

File: charts/Chart.yaml [hunk 3]
```diff
@@ -1,7 +1,17 @@
...
+home: https://github.com/Opster/opensearch-k8s-operator
+icon: https://opster.com/wp-content/uploads/2022/04/Opster_Logo_Rectangle_Dark.svg
+maintainers:
+  - email: idan@opster.com
+    name: Idan levi
+  - email: ori.g@opster.com
+    name: Ori Goldstein
+sources:
+  - https://github.com/Opster/opensearch-k8s-operator
+version: "1.1"
```
Comment (swoehrl-mw) [inline charts/Chart.yaml:14]: This should correspond to the version of the operator

**Review by swoehrl-mw — CHANGES_REQUESTED** [review]

File: charts/Chart.yaml (hunk 3 above)
Comment (idanl21) [reply charts/Chart.yaml]: changed

File: charts/templates/opensearch-operator-system-namespace.yaml (hunk 1 above)
Comment (idanl21) [reply charts/templates/opensearch-operator-system-namespace.yaml]: deleted

**Pull request closed without merging**